reorganize-2019-01.md

```diff
@@ -0,0 +1,86 @@
+# Reorganization for rstudio::conf 2019-01
+
+The outline below is for day 1 only.
+
+## 01) Two Ways to Teach
+
+-   Introductions
+-   Comparison with movie/play (add comedy improv)
+-   Tell participants there's lots of research (much of which people don't know/act on)
+-   Schedule (for both days)
+-   Survey distinguishing teacher-centered from learner-centered
+    -   Explain that active learning creates a real-time feedback loop in class
+-   Introduce cognitive architecture diagram (revisit it several times)
+    -   Teaching is "load into short-term memory and keep it there long enough to be transferred"
+-   Bloom's Taxonomy
+    -   Use revised version from early 2000s
+    -   Google isn't enough if you don't know what to search for or how to recognize a useful answer when you find one
```

Comment:
FWIW, I added awareness to the bottom o the taxonomy for this reason (and to humbly point out that it is perhaps the most realistic objective for a short workshop).